### PR TITLE
Feature/7 create base markup

### PR DESCRIPTION
**Pull request by mrgumorist**
Not cool

### CODE REVIEW

File: BobManager.Client/client/src/app/app.component.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { Component } from '@angular/core';
-
+import {FooterComponent} from './footer/footer.component'
```
Comment (Warefoxe) [inline BobManager.Client/client/src/app/app.component.ts]: format

Comment (MickeyLeet) [inline BobManager.Client/client/src/app/app.component.ts]: format pls

File: BobManager.Client/client/src/app/app.component.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+import {HomeComponent} from './home/home.component'
```
Comment (Sarhsian) [inline BobManager.Client/client/src/app/app.component.ts]: Format, please gay

File: BobManager.Client/client/src/app/app.component.ts (hunk 1 above)
Comment (mrgumorist) [reply BobManager.Client/client/src/app/app.component.ts]: resolved

File: BobManager.Client/client/src/app/app.component.ts (hunk 2 above)
Comment (mrgumorist) [reply BobManager.Client/client/src/app/app.component.ts]: resolved

**Review by MickeyLeet — APPROVED** [review]
approve

**Review by Sarhsian — APPROVED** [review]
Approve, gay

**Review by Warefoxe — APPROVED** [review]
Approve

**Pull request merged**